src/azure-cli/azure/cli/command_modules/role/_breaking_change.py

```diff
@@ -0,0 +1,8 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.breaking_change import register_argument_deprecate
+
+register_argument_deprecate('role assignment list', '--include-classic-administrators')
```

Comment:
If I register deprecation for `--all`, `register_argument_deprecate` fails: https://github.com/Azure/azure-cli/issues/31047

Reply:
Should work fine after #31057 merged.
![image](https://github.com/user-attachments/assets/0037196c-2a5e-4958-8736-2ed5476a9cb0)
![image](https://github.com/user-attachments/assets/4f2a512a-6e17-4beb-a36f-e94c0977aeb6)